### DISCUSSION TITLE
[@types/ramda] 0.28 merge removed

### DISCUSSION THREAD

**Discussion by levino** [DefinitelyTyped/DefinitelyTyped#60269] [Issues with a @types package]
`R.merge` was removed in [`ramda@0.28`](https://github.com/ramda/ramda/issues/3218) but the types still show it as present. Leads to nasty bugs. Potentially the other removals are also still present in the types.

[a comment by an automated account is not shown]

**Comment by levino** [comment]
I can open a MR but will wait a bit to gather some feed back.

**Reply by adispring** [reply]
Welcome to make a MR. There are some other apis that should be removed from @types/ramda, such as `contains`,  `composeP` `composeK`.
